### PR TITLE
Test warnings in Expert Partitioner when too small root, /boot/zipl and PReP are used

### PR DESCRIPTION
**Pull request by OleksandrOrlov**
- Related ticket: https://progress.opensuse.org/issues/80840
- Verification runs:
   - SLE: https://openqa.suse.de/tests/overview?build=124.5_oorlov&groupid=96&distri=sle&version=15-SP3
   - opensuse: https://openqa.opensuse.org/tests/1588622

### CODE REVIEW

File: schedule/yast/btrfs/btrfs+warnings_opensuse.yaml [hunk 1]
```diff
@@ -46,10 +49,25 @@ test_data:
...
+            mounting_options:
+              should_mount: 1
+              mount_point: /
   errors:
     <<: !include test_data/yast/btrfs/common/btrfs+warnings_errors.yaml
   warnings:
     <<: !include test_data/yast/btrfs/common/btrfs+warnings_warnings.yaml
     no_boot: A partition of type BIOS Boot Partition is needed to install the bootloader
     bios_boot_small_for_bootloader: A partition of type BIOS Boot Partition is needed to install the bootloader
     uefi_small: A partition of type BIOS Boot Partition is needed to install the bootloader
```
Comment (jknphy) [inline schedule/yast/btrfs/btrfs+warnings_opensuse.yaml:70]: if these three are always the same, why not just have one key?

File: tests/installation/partitioning/warning/zipl_small.pm [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
```
Comment (jknphy) [inline tests/installation/partitioning/warning/zipl_small.pm:3]: could you please update copyright for all files needed?

**Comment by jknphy** [comment]
In the first comment, let's delete the template line so the PR will not be taken as WIP by mistake (like is some draft or something). I fell into that :)

File: schedule/yast/btrfs/btrfs+warnings_opensuse.yaml (hunk 1 above)
Comment (OleksandrOrlov) [reply schedule/yast/btrfs/btrfs+warnings_opensuse.yaml]: Good point! I've replaced all these keys with the one key. I'll re-run the VRs.

File: tests/installation/partitioning/warning/zipl_small.pm (hunk 2 above)
Comment (OleksandrOrlov) [reply tests/installation/partitioning/warning/zipl_small.pm]: Done.

**Pull request merged**